lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc

```diff
@@ -66,11 +66,19 @@ function dosomething_reportback_build_reportbacks_query($params = array()) {
   }
 
   if (isset($params['runs'])) {
-    if (is_array($params['runs'])) {
```

Comment:
I'm still a little lost on why we remove this line -- can dive a little deeper into the functions but @weerd curious if this is a quick one to review?